spec/routific_spec.rb

```diff
@@ -125,6 +129,24 @@
         expect(route).to be_instance_of(RoutificApi::Route)
       end
     end
+
+    describe "#get_route_async" do
+      before do
+        job = nil
+        set_visit_and_vehicle routific
```

Comment:
Should we clear the existing visits and vehicle first?

Reply:
nevermind, i just read you comment....

Reply:
It seems that the :visits and :fleet attributes are cleared by the `describe`. So that's fine.